src/swagger-specs/destinations.yaml

```diff
@@ -130,88 +368,624 @@ paths:
-          schema:
-            type: string
+        - name: id
+          in: path
+          description: "The ID of the connection you are looking to delete." 
           required: true
-        - in: header
-          name: Accept
-          description: "An optional content negotiation header which allows you to use a specific version of the API. If you don't use it, the HTTP responses fall back to the latest version of the API."
           schema:
             type: string
-            default: "application/json; version=1"
-          required: false
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/XApiKey'
+        - $ref: '#/components/parameters/XImsOrgId'
+        - $ref: '#/components/parameters/XSandboxId'
+        - $ref: '#/components/parameters/Accept'
+      responses:
+        '202':
+          description: A successful response returns HTTP status 202 (No Content) and a blank body. You can confirm the deletion by attempting a lookup (GET) request to the connection. The API will return an HTTP 404 (Not Found) error, indicating that the connection has been deleted.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '401':
+          description: The token provided in the Authorization header is invalid.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              example:
+                error_code: 401013
+                message: "Oauth token is not valid"
+        '403':
+          description: You are not authorized to perform this operation 
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '404':
+          description: The specified base connection ID was not found. The resource may have been deleted, or the requested path was entered incorrectly.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              example:
+                errorMessage: "connections not found with id = [ca5c6e9a-8559-4a08-97a5-f15b756da600-]"
+                errorDetails: "connections not found with id = [ca5c6e9a-8559-4a08-97a5-f15b756da600-]"
+        '500':
+          description: An Internal error occurred.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+      # externalDocs:
+      #   description: Learn more about user operations provided by this API.
+      #   url: http://api.example.com/docs/user-operations/
+
+  /sourceConnections:
+    get:
+      tags:
+       - Source connections
+      summary: Retrieve a list of destination source connections
```

Comment:
"destination source connections" seems like it might be misleading/confusing. It's a bit more of a mouthful, but maybe "source connections for destinations" is a bit less ambiguous?